### DISCUSSION TITLE
Circular link for Columbia Transit

### DISCUSSION THREAD

**Discussion by ChasBelov** [transitland/transitland#374] [Q&A] [answered]
If there's no URL in a routes.txt file entry, TransitLand creates a link anyway which goes to the page itself.

On the [TransitLand page for the Columbia College Shuttle](https://www.transit.land/routes/r-9qf2-columbiacollegeshuttle), the URL link icon links to the same page in a new tab.

Also happens on the [TransitLand page for 71X Everson/Nooksack/Sumas&Cordata](https://www.transit.land/routes/r-c28y-71x)

**Comment by irees** [comment, marked as answer]
Hi - thank you for reporting this issue. It looks to have been resolved by an (unrelated) recent improvement to the routes page.
